experimenter/experimenter/experiments/models.py

```diff
@@ -392,7 +392,7 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
         blank=True,
         null=True,
     )
-    firefox_labs_description_links = models.JSONField[dict[str, str]](
+    firefox_labs_description_links = models.TextField(
```

Comment:
Wait what's the motivation for switching to textfield?  We wanna put JSON in it and validate that it's JSON but keep it as str?  We use JSONField elsewhere?

Reply:
JSONField /must/ be valid JSON when saving. We're currently attempting to save arbitrary text to it and its becoming `json.dumps({ whatever })` instead of `{ whatever }`.